playbooks/files/rax-maas/plugins/keystone_api_local_check.py

```diff
@@ -34,7 +34,7 @@ def check(args, auth_details):
         port=args.port,
         ip=args.ip
     )
-    if auth_details['OS_AUTH_VERSION'] == '2':
+    if auth_details.get('OS_AUTH_VERSION', None) == '2':
```

Comment:
nit: `.get` defaults to returning `None`, so that's redundant

in other words, you can just do: `auth_details.get('OS_AUTH_VERSION') == '2'`